src/wp-admin/themes.php

```diff
@@ -133,8 +133,8 @@
 		'<p>' . __( 'From this screen you can:' ) . '</p>' .
 		'<ul><li>' . __( 'Hover or tap to see Activate and Live Preview buttons' ) . '</li>' .
 		'<li>' . __( 'Click on the theme to see the theme name, version, author, description, tags, and the Delete link' ) . '</li>' .
-		'<li>' . __( 'Click Customize for the current theme or Live Preview for any other theme to see a live preview' ) . '</li></ul>' .
-		'<p>' . __( 'The current theme is displayed highlighted as the first theme.' ) . '</p>' .
+		'<li>' . __( 'Click Customize for the currently active theme or Live Preview for any other theme to see a live preview' ) . '</li></ul>' .
```

Comment:
You knew this was coming 😂

What made you make this one different to the others?

Reply:
yeah, better to stick with the same wording :)